backend/src-common/src/main/resources/RejectedCREmailemplate.html

```diff
@@ -0,0 +1,21 @@
+<!--
+  ~ Copyright Siemens AG, 2020. Part of the SW360 Portal Project.
+  ~
+  ~ This program and the accompanying materials are made
+  ~ available under the terms of the Eclipse Public License 2.0
+  ~ which is available at https://www.eclipse.org/legal/epl-2.0/
+  ~
+  ~ SPDX-License-Identifier: EPL-2.0
+-->
+<!doctype html>
+<html>
+  <body>
+	<p>*** This is an automatically generated email, please do not reply. ***</p>
+	<p>Dear SW360-user,
+	    Your clearing request with id: <b>%s</b> for the project: <b>%s</b> has been rejected by the clearing team.

```

Comment:
looks same like `ClosedCREmailTemplate.html` . Could be merged using parameter for `rejected/closed`